### ISSUE TITLE
[Issue] Non-detection of user plugin configuration `.lua` files in **LazyVim**

### ISSUE THREAD

**Issue by pidgeon777** [polirritmico/telescope-lazy-plugins.nvim#14]
In a previous version of the **LazyVim** plugin, `telescope-lazy-plugins.nvim` was capable of recognizing files located in the user's `extras` folder. Specifically, the root path is as follows:

```
C:\Users\<USERNAME>\AppData\Local\nvim\lua\plugins\extras
```

This directory contains various subfolders, each housing `.lua` files. The `.lua` files in this directory are plugin activation and configuration files, formatted in a manner supported by **LazyVim**. 

![image](https://github.com/user-attachments/assets/fc3eb9d0-02a9-4538-a163-bf3a326d5475)

The `___cfg.lua` files shown above are stored in the user's parent directory, specifically at:

```markdown
C:\Users\<USERNAME>\AppData\Local\nvim\lua\plugins
```

These files are correctly indexed by `telescope-lazy-plugins.nvim`, as shown in the screenshot.

However, It is unclear to me whether the absence of the files located in the user `plugins` subfolder from the `telescope-lazy-plugins.nvim` list is a bug, or if a specific configuration is required to display them, possibly due to a plugin update. 

Here is my current configuration:

```lua
return {
  {
    "polirritmico/telescope-lazy-plugins.nvim",
    dependencies = {
      "nvim-telescope/telescope.nvim",
      opts = function(_, opts)
        -- HACK: We define some functions here, to be called when the corresponding action is triggered.
        -- The requires are done inside those functions. By not directly associating the requires to the
        -- corresponding mappings, we avoid the extensions not to be lazy-loaded.
        local open_repo_url = function(bufnr)
          require("telescope").extensions.lazy_plugins.actions.open_repo_url(bufnr)
          require("telescope.actions").close(bufnr)
        end
        local open_repo_dir = function(bufnr)
          require("telescope").extensions.lazy_plugins.actions.open_repo_dir(bufnr)
        end
        local open_repo_live_grep = function(bufnr)
          require("telescope").extensions.lazy_plugins.actions.open_repo_live_grep(bufnr)
        end

        local my_opts = {
          lazy_plugins = {
            name_only = false,
            mappings = {
              ["i"] = {
                -- FIXME: Can't remove default mappings. Open an issue.
                ["<C-g>x"] = nil,
                ["<C-g>r"] = nil,
                ["<C-g>l"] = nil,
                -- WARN: We don't do like this:
                -- ["<C-h>"] = require("telescope").extensions.lazy_plugins.actions.open_repo_url,
                -- otherwise 'lazy_plugins' extensions won't be lazy-loaded.
                -- By calling a function instead, it will be lazy-loaded.
                ["<C-h>"] = open_repo_url,
                ["<C-o>"] = open_repo_dir,
                ["<C-z>"] = open_repo_live_grep,
              },
            },
          },
        }

        opts.extensions = vim.tbl_deep_extend("force", opts.extensions or {}, my_opts)

        return opts
      end,
    },
    -- event = "VeryLazy",
    cmd = "Telescope lazy_plugins",
    keys = {
      { ",u", "<Cmd>Telescope lazy_plugins<CR>", desc = "Lazy Plugins" },
    },
    enabled = lvim.builtin.telescope_lazy_plugins.active,
  },
}
```

**Comment by polirritmico** [comment]
Hi again, thanks for all the bug report! It's really appreciated.

Normally, lazy.nvim does not automatically import subfolders, so they must be added manually:

```
{
  -- ... etc.
  { import = "plugins" },
  { import = "plugins.extras" },
},
```

But you are saying that this was working before without this (lazy < v11), so LazyVim should be adding them somewhere.

I'm going to look into this.

**Comment by pidgeon777** [comment]
Thanks or the quick reply! I enable or disable the extras with `LazyExtras`, then press `x` on the plugin I want to toggle:

![image](https://github.com/user-attachments/assets/b9b86653-856f-42b8-add7-dceb16222e52)

The plugins marked orange named `User` are the ones located inside the `C:\Users\<USERNAME>\AppData\Local\nvim\lua\plugins\extras` folder. These are detected by `LazyVim`, but not by `telescope-lazy-plugins.nvim`.

**Comment by polirritmico** [comment]
Fixed by edc849b. Could you confirm?

**Comment by pidgeon777** [comment]
It seems so! Great work.
